components/dashboard/src/teams/teams-context.tsx

```diff
@@ -56,17 +54,6 @@ export function useTeams(): Team[] | undefined {
     return teams;
 }
 
-export function useBillingModeForCurrentTeam(): BillingMode | undefined {
```

Comment:
I coincidentally had made a version of this using react-query, so updated consumers to use that. This way it stays fresh as the org changes, and shifts more data into that query cache.